ddp-workspace/projects/ddp-dsm-ui/src/app/tissue-list/tissue-list.component.html

```diff
@@ -129,31 +129,35 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
             <ng-container *ngFor="let savedFilter of this.savedFilters; let i=index">
               <tr *ngIf="role.userID() === savedFilter.userId">
                 <td>
-          <span style="font-size: 15px; padding-right: 30px;">
-            {{savedFilter.filterName}}
-          </span>
-                  <button class="float-right" color="warn"
-                          mat-icon-button (click)="deleteView(savedFilter)">
-                    <mat-icon>
-                      <i class="fas fa-trash-alt fa-sm"></i>
-                    </mat-icon>
+                  <button class="float-right"
+                          mat-icon-button [color]="selectedFilterName === savedFilter.filterName ? 'accent' :'black'"
+                          (click)="selectFilter(savedFilter)">
+                     <span style="font-size: 15px;">
```

Comment:
I believe you can get rid of that span and set the style for the button in the proper scss file.